### ISSUE TITLE
Separate custom text from display update

### ISSUE THREAD

**Issue by sragia** [WeakAuras/WeakAuras2#1129]
**Is your feature request related to a problem? Please describe.**
Basically quite few people have asked me how to make sure that font stays the same after update and I have to say to them to not update Display. Issue is that it really restricts my (as WA author) ability to modify custom text. If I tell them to not update Display they won't get my fixes/new features shown, that could potentially even break aura if I lets say change some aura_env var I have defined in trigger or wherever. Looking at Display tab to me this is the main offender as something that WA author could change as everything else 99% of time is just visual and something that user is most likely to change.

**Describe the solution you'd like**
Extra toggle in update as I don't think it could be added to something else. Something like Display Texts or something. Also probably actual texts should be included too (text1 text2) as maybe with an update custom text is added not modified, so without %c user won't see it.

**Describe alternatives you've considered**
I mean alternative as author obviously could be %c1 %c2 in both texts (icon) and custom function
```lua
function()
  return aura_env.text1, aura_env.text2
end
```
and change them in trigger or somewhere else (because you aren't always using custom trigger). And you probably will never need to update it.
but that's just workaround that complicates things

**Comment by emptyrivers** [comment]
That seems like too narrow of an option. I see the problem, but we don't really have the room in that tooltip to add a category for every single addition like this.

I wouldn't be opposed to separating all custom scripts into one category (calling it `Scripts` or `Custom Code`), though.

**Comment by sragia** [comment]
Yeah that's true for some auras tooltip starts to get a bit too crowded and overwhelming for user. 
Custom Code update option probably is the best solution here. Especially when custom options exist and you don't really need/have auras with options in Init or some other custom code block.

**Comment by emptyrivers** [comment]
@Stanzilla @InfusOnWoW @mrbuds do any of you have thoughts (specifically, objections) on this?

**Comment by mrbuds** [comment]
Sound good, i don't see it adding regressions.
If that can make things easier for transmissions, wago, and update tooltip for end-user it's nice.

**Comment by Stanzilla** [comment]
I don't quite understand what your suggestion has to do with the original problem somehow. In my understanding the problem is that users want to keep their fonts but authors might want to change from %p to %c etc and those changes won't stick if people don't update the display category

**Comment by emptyrivers** [comment]
My understanding was that the issue refers specifically to the code itself, and not the text which pulls from custom code.

I see your point, though.

**Comment by Stanzilla** [comment]
Oh yeah, that's a bit of a pickle since what you proposed would lead to the %c code being preserved but then we could still into the problem I mentioned. I have no problem with the content of %c being moved from display to custom code but it might be not very transparent for users since technically it's still *inside* the display tab

**Comment by emptyrivers** [comment]
Yup, I'm becoming increasingly convinced that the update tooltip just isn't sufficient on its own. There are just too many corner cases which erode its usefulness.

**Comment by Stanzilla** [comment]
When first talking about it we were considering a setup wizard like experience for importing and updating so we could get rid of the tooltip once and for all, maybe when we go that route we can offer finer grained update options as well. I just really don't want to make it too complicated, the few options we currently have are already too much for some
